hack/run-openshift

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+#
+# Test the MongoDB image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+set -exo nounset
+
+function prepare_oc_client() {
+  # Prepare oc command
+  if [[ ! -d ../openshift-client/ ]]; then
+    OC_RELEASE=$(curl -s https://api.github.com/repos/openshift/origin/releases | grep "browser_download_url" | grep "openshift-origin-client" | grep "linux-64bit" | sort | tail -n 1 | sed -e 's|"browser_download_url": ||' | tr -d ' "')
```

Comment:
i'm getting a rate limiting failure hitting that url, i suspect that may cause flake issues for you...